org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/AbstractRemoteService.java

```diff
@@ -27,6 +29,8 @@
  */
 abstract class AbstractRemoteService implements IDisconnectable, Closeable {
 
+	protected final static Logger logger = LoggerFactory.getLogger(AbstractRemoteService.class);

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)